Merging dictchanges_CMSSW94X branch into master

hsert opened the pull request:
Dear all,
This pull request shows the changes on the dictchanges_CMSSW94X branch to be merged into the master. I was surprised with the commits from early-mid 2017, since I thought we branched off close to the end of the year. If you have any explanation for it, I would be glad to hear those.
Thanks for the review in advance.
Best,
Hale

thomas-mueller committed:
Revert "Merge pull request #51 from KappaAnalysis/getByToken"

This reverts commit 12ab11cc14515a0fba3c733bf8bc0683574f049d, reversing
changes made to c74bf50b9771436e95e6e2082eaef0ee4bebd30a.

Conflicts:
	Producers/interface/KGenInfoProducer.h
	Producers/interface/KMuonProducer.h

thomas-mueller committed:
KInfo.h: modify unreasonable assertion

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

Conflicts:
	DataFormats/interface/KInfo.h

thomas-mueller committed:
Revert "Revert "Merge pull request #51 from KappaAnalysis/getByToken""

This reverts commit fe54fcb5d7986018e8d5590f56d41690ee88de4e.

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

thomas-mueller committed:
KInfo.h: remove check, whether weights are within certain/arbitrary range

thomas-mueller committed:
KInfo.h: getLheWeightNamesMap returns again the requested name (not the mapped internal name as before)

thomas-mueller committed:
Merge remote-tracking branch 'origin/dictchanges' into getByToken

thomas-mueller committed:
LinkDef.h: remove file which is not needed any more

thomas-mueller committed:
KInfoProducer.h, KInfo.h: create/fill/save Runs tree based on the same principle as the Lumis tree

thomas-mueller committed:
KElectronProducer.h, KMuonProducer.h: minor changes (comments, indentaion)

thomas-mueller committed:
KLeptonPairProducer.h: change to getByToken

thomas-mueller committed:
KExtendedTauProducer.h: change to getByToken

thomas-mueller committed:
KPatTauProducer.h: change to getByToken

thomas-mueller committed:
KCaloJetProducer.h, KFilterSummaryProducer.h: added comments

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

thomas-mueller committed:
Merge remote-tracking branch 'origin/dictchanges' into getByToken

greyxray committed:
Last update of Kaon production, sync with getByToken

greyxray committed:
Merge remote-tracking branch 'origin/dictchanges' into dictchanges

greyxray committed:
removing log output (1)

greyxray committed:
Test script is cleaned up

greyxray committed:
KPatTau: revert non-working getByToken

Commit:
Merge pull request #53 from KappaAnalysis/getByToken

Get by token (part 3)

greyxray committed:
Remove all Kaon information

greyxray committed:
TESTS: enable more output

thomas-mueller committed:
KGenInfoProducer.h: fix LHE weight regex, keyword in weightgroup can be either "type" or "name"

thomas-mueller committed:
kSkimming_run2_cfg.py: improve LHE settings

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

greyxray committed:
checkout script 8: comment the dublicating package checkout

Commit:
Update test_build.sh

add wget installation

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

Conflicts:
	Skimming/higgsTauTau/kSkimming_run2_cfg.py

thomas-mueller committed:
Merge remote-tracking branch 'origin/dictchanges' into dictchanges

greyxray committed:
clean up

thomas-mueller committed:
kSkimming_run2_cfg.py: store LHE weights and particles only for Z/H->tautau events and LFV samples

thomas-mueller committed:
Merge remote-tracking branch 'origin/dictchanges' into dictchanges

thomas-mueller committed:
KPatTauProducer.h: fixed clean up commit

greyxray committed:
fix the muon configuration

greyxray committed:
fix config for electrons

greyxray committed:
Add exceptions calls

hsert committed:
added missing parts for srcMuonIsolationPF pset and some more updates

hsert committed:
Merge branch 'dictchanges' of https://github.com/KappaAnalysis/Kappa into dictchanges

Conflicts:
	Producers/interface/KMuonProducer.h

stwunsch committed:
Let git ignore vim swap files

stwunsch committed:
Whitespace fixes on datasets.json

stwunsch committed:
Remove 2017 datasets from database

Conflicts:
	Skimming/data/datasets.json

stwunsch committed:
Readd 2017 datasets (data only) up to D v1

stwunsch committed:
Add missing global tags to 2017 data

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

Conflicts:
	Skimming/higgsTauTau/kSkimming_run2_cfg.py

stwunsch committed:
Add Summer17 MC samples: DYJetsToLL, WJetsTLNu, TT

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

perahrens committed:
Added EmbeddingRun2017B MuTau dataset

thomas-mueller committed:
SkimManager.py: simplify calls of the crab_cmd function, version with pool was lacking a proper timeout

thomas-mueller committed:
datasetsHelperTwopz.py: sort nick lists for a more deterministic behaviour

thomas-mueller committed:
datasets.json: remove non-existing sample

thomas-mueller committed:
kSkimming_run2_cfg.py: minor change

thomas-mueller committed:
kSkimming_run2_cfg.py: put lines to selection few events on a safer place

thomas-mueller committed:
SkimManager.py: enabled forced submission of jobs in exception state

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

thomas-mueller committed:
KPatTauProducer.h: fix indentation

thomas-mueller committed:
KTau.h, KPatTauProducer.h: added refitted a1 properties

thomas-mueller committed:
KTau.h: added resonancePdgId function

thomas-mueller committed:
Merge remote-tracking branch 'origin/master' into dictchanges

greyxray committed:
Fix config file to process in 9XX

greyxray committed:
add more comments to deep-debug - run with verbosity 3

greyxray committed:
update on configuretions

greyxray committed:
temp commit to make it run on 2017. This commit has to be reverted once the issue will be found and fix

hsert committed:
some of the missing MC 2017 samples are added

hsert committed:
missing 2017 data samples are added

hsert committed:
missing tags are added for summer16 DY samples (moriond17)

thomas-mueller committed:
KTuple_cff.py, kSkimming_run2_cfg.py: fix trigger objects src for Fall15/CMSSW7 samples

thomas-mueller committed:
kSkimming_run2_cfg.py: fix trigger objects src for Spring16/CMSSW8 samples

hsert committed:
updates in datasets and a copy of kSkimming_2016 is added for 2017 samples

Commit:
update the recipe for the tau

and clean the comments a bit

Commit:
fix checkout8

remove the conflicting package - the diff should be implemented in new tau backport.
this is a test commit

hsert committed:
Some of the 2017 datasets are added

thomas-mueller committed:
registerDatasetHelper.py, DatasetManager.py: fun with code duplication

thomas-mueller committed:
registerDatasetHelper.py, DatasetManager.py: adjustments for JHU samples

thomas-mueller committed:
registerDatasetHelper.py, DatasetManager.py: fun with code duplication (part 2)

thomas-mueller committed:
datasets.json: only re-formatting; this file should be edited with DatasetManager.py

thomas-mueller committed:
registerDatasetHelper.py, DatasetManager.py: minor change for JHU generators

thomas-mueller committed:
datasets.json: add signal samples used in AN-2017/212 for CP analysis

thomas-mueller committed:
datasets.json: remove old/superceeded JHU CPmixing samples

Commit:
fixing checkoutCmssw80xPackagesForSkimming.sh

test commit.
change the checkout order in order to compile and resolve conflicts in favour of cms-tau-pog:CMSSW_8_0_X_tau-pog_tauIDOnMiniAOD-legacy-backport-81X

Commit:
Update checkoutCmssw80xPackagesForSkimming.sh

apparently(my suspicion), the tau pog old recipe was checkout and committed in macewindu009 topic, so now no need to check out the new TAU pogs recipe ¯\_(ツ)_/¯

thomas-mueller committed:
datasets.json: added missing JHU ggH samples

thomas-mueller committed:
datasets.json: added temporarily xsec=1.0 for new JHU CP signal samples

thomas-mueller committed:
kSkimming_run2_cfg.py: save LHE particles also in JHU CP signal samples

hsert committed:
the modified one is copied from analysis environment

hsert committed:
Revert "the modified one is copied from analysis environment"

This reverts commit 2faff66dc9f9d533ec4c3babe7f2c58ef7e4f659.

thomas-mueller committed:
dataset.json: added xsecs for JHU CP samples taken from https://twiki.cern.ch/twiki/bin/viewauth/CMS/Run2MCProductionforHiggsProperties

DatasetManager.py --overwrite --nicks GluGluH2JetsToTauTauM125CPmixingmaxmixJHU_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*29169.2\" | bc`}"
DatasetManager.py --overwrite --nicks GluGluH2JetsToTauTauM125CPmixingpseudoscalarJHU_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*14397.13\" | bc`}"
DatasetManager.py --overwrite --nicks GluGluH2JetsToTauTauM125CPmixingsmJHU_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*14583.61\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0L1M125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*0.000208309\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0L1f05ph0M125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*2861.213\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0MM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*10909.54\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0Mf05ph0M125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*1937.15\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0PHM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*13102.71\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0PHf05ph0M125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*2207.73\" | bc`}"
DatasetManager.py --overwrite --nicks VBFHiggs0PMM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgenv6 --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*968.674\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0L1UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*0.1123491\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0L1fWH05ph0UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*25302.37\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0MUndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*2028656\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0MfWH05ph0UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*61997.07\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0PHUndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*3106339\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0PHfWH05ph0UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*16486.68\" | bc`}"
DatasetManager.py --overwrite --nicks WHiggs0PMUndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*30998.54\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0L1UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*33652.46\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0L1fZH05ph0UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*6852.307\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0MUndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*434763.7\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0MfZH05ph0UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*18044.72\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0PHUndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : `echo \"scale=12; 0.001*0.0632*713123\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0PHfZH05ph0UndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*4258.966\" | bc`}"
DatasetManager.py --overwrite --nicks ZHiggs0PMUndecayedM125_RunIISummer16MiniAODv2_PUMoriond17_13TeV_MINIAOD_JHUgen --addentry "{\"xsec\" : 0`echo \"scale=12; 0.001*0.0632*9022.36\" | bc`}"

greyxray committed:
add the miissing 2017 MCv1

greyxray committed:
add the missing tags

greyxray committed:
styling datasetsHelperTwopz.py

greyxray committed:
remove the annoying trailing spaces from datasets.json and automatic script

Commit:
Fix to make skimming work for CMSSW>9_4

greyxray committed:
basic setup. fix the compilation issue

hsert committed:
17Nov Rereco data samples and some Fall17-MC samples are added

greyxray committed:
style:remove trailing spaces and add the tabs

greyxray committed:
style: fix the tabs

greyxray committed:
make the TransientTracks compilable again

greyxray committed:
minor cleanup

greyxray committed:
NANOPROPOSAL

greyxray committed:
style: switch to conventional modules inclusion

greyxray committed:
enable the initial configure for thr VertexRefit module

greyxray committed:
bringing back the basick TT builder...

hsert committed:
more 94X samples are added

hsert committed:
more 94X samples are added

greyxray committed:
step to bring back the DV refits

greyxray committed:
very dirty version of refitter

greyxray committed:
style: clean up KElectronProducer.h

greyxray committed:
make KMuonProducer.h standartised and working

greyxray committed:
bring back the standartised SV refit to the TAU-related producers

hsert committed:
Merge remote-tracking branch 'origin/dictchanges_CMSSW94X' into dictchanges_CMSSW94X

hsert committed:
one MC94X dataset is added

hsert committed:
missing Xsections are added for Summer17

hsert committed:
some new fall17 samples are added

hsert committed:
Electron and Tau ID are updated for 94X samples

hsert committed:
ttbar Fall17 datasets are added

hsert committed:
Uncompleted Wjets Fall17 datasets are added manually

hsert committed:
Fall17 datasets: some corrections

hsert committed:
Fall17 dataset, correction on WJet dataset

hsert committed:
generatorWeights are added for MC samples

dwolfschlaeger committed:
Make Skimmanager initialize without specifying a whitelist.

greyxray committed:
add checkoutCmssw94xPackagesForSkimming.sh

greyxray committed:
add the downloadting of the script used for accessing of tau MVA

greyxray committed:
Update checkoutCmssw94xPackagesForSkimming.sh

remove unnecessary lines

hsert committed:
some cleanings and additions

greyxray committed:
remove warnings during compilation

greyxray committed:
update checkoutCmssw94xPackagesForSkimming.sh

greyxray committed:
Add DeepCSV b-tagging to A(4;8)PF(CHS;PUPPI) jets

greyxray committed:
Jets: store PatJet hadron and parton flavours. HTT uses hadronFlavour

greyxray committed:
checkout script update

greyxray committed:
add the L1 taus to the Kappa format and skims

greyxray committed:
EventWeightCountProducer: fix the indentation

greyxray committed:
add the missing producer to commit 59c6302d93ff961af4c7ec7777a952d8276fe23d

greyxray committed:
update with new cutbase ID 2017 and remove IDs of 2015 where there are newer

greyxray committed:
Electrons: store  cutbase ID 2017

greyxray committed:
store PF met

greyxray committed:
add notes&comments

greyxray committed:
Update the PSet with new access to tau MVA

greyxray committed:
mark as obsolete KPatTaus_run2_cff.py before deletion

greyxray committed:
KBaseMultiProducer.h: add more senseble warning

greyxray committed:
DatasetManager::copy_xsec_from_campaign:  copy cross-sections process-wise from previous campaigns

hsert committed:
NonIso ElectronID is added

hsert committed:
upgraded to Cmssw_9_4_4 and option for the scaling and smearing for electron is added

hsert committed:
Revert "upgraded to Cmssw_9_4_4 and option for the scaling and smearing for electron is added"

This reverts commit db7a64bc9043968541e99aff5050d0d968e9fa39.

hsert committed:
Revert "Revert "upgraded to Cmssw_9_4_4 and option for the scaling and smearing for electron is added""

This reverts commit b76addd371c6c265c3b1e7ee7de61849ab474ea5.

greyxray committed:
KEleProducer: increase readability

greyxray committed:
2016: don't include L1 taus to the skim

hsert committed:
a reference is added for new tauID implementation

greyxray committed:
Add the preprocessing to support cmssw8 and cmssw9

greyxray committed:
Fix the processing of 2016 samples; for some reason the line  creates problem if the 2015 electron IDs are not included

hsert committed:
GT update, and 2 single top MC sample is added

hsert committed:
Merge remote-tracking branch 'origin/dictchanges_CMSSW94X' into dictchanges_CMSSW94X

hsert committed:
tags/tagvalues are added for new St samples

hsert committed:
a few more changes on 2017_v2 datasets

hsert committed:
final updates on dataset.json file before the new skim

hsert committed:
update to make it run in the submission, otherwise it can not find the files under external folder

hsert committed:
updates to make SkimManager run with kSkimming_2017_cfg file

hsert committed:
Merge branch 'dictchanges_CMSSW94X' of github.com:KappaAnalysis/Kappa into dictchanges_CMSSW94X

hsert committed:
GT for 2017 data is updated

greyxray committed:
for data 2017 <pat::met> is produced by RECO step

hsert committed:
An option to run over an incomplete dataset is added

greyxray committed:
Add the Automatic file splitting. Handle unknown errors differently and allow to ignore them unless it's submittion

hsert committed:
generator weights for 2017 MCs are updated

hsert committed:
Merge remote-tracking branch 'origin/dictchanges_CMSSW94X' into dictchanges_CMSSW94X

hsert committed:
A correction to generator weights: change from string to float.

greyxray committed:
Eddit 2016 config to point that the tau id access is obsolete

greyxray committed:
Add the pprint import

greyxray committed:
Fix the list of wanted MVA for 2017 data

hsert committed:
TauID: stored names of the TauIDs are changed

greyxray committed:
Fix the Automatick splitting for submition

greyxray committed:
some PSet adjustments

greyxray committed:
makes no sence to limit floatDiscriminatorNames by the same amount as binaryDiscriminatorNames

greyxray committed:
keep the 0.3 cone in the 2017 skims

greyxray committed:
Add debug printout for Taus

jd550179 committed:
added gluonfusion powheg file to the dataset

jd550179 committed:
added tagvalues

jd550179 committed:
added VBFHToTauTauM125_RunIIFall17MiniAODv2_PU2017_13TeV_MINIAOD_powheg-pythia8 to datasets.json

greyxray committed:
Merge remote-tracking branch 'origin/master' into dictchanges_CMSSW94X

Summary about some important for the future PR points:
* Producers/interface/KMuonProducer.h:are commented out!!!
    * cEvent->getByToken(this->tokenRefitVertices, this->RefitVertices);
    * KTrackProducer::fillIPInfo(*in.globalTrack(), out.globalTrack,
    * *RefitVertices, trackBuilder.product());
* KElectronProducer.h
    * in future this commit should be reverted:
    * https://github.com/KappaAnalysis/Kappa/commit/1512fa704c1ea67f1e61c407ae90591f97ca71fa#diff-e88fd359353773927a23f1e3ea1b43ab
    * cEvent->getByToken(this->tokenRefitVertices, this->RefitVertices);
    * // KTrackProducer::fillIPInfo(*in.gsfTrack(), out.track,
    * *RefitVertices, trackBuilder.product());
* Producers/interface/KPatTauProducer.h :
    * // KTrackProducer::fillIPInfo(*packedLeadTauCand->bestTrack(),
    * out.track, *RefitVertices, trackBuilder.product());
    * //edm::ESHandle<TransientTrackBuilder> trackBuilder =
    * edm::ESHandle<TransientTrackBuilder>();
    * // cEvent->getByToken(this->tokenRefitVertices,
    * this->RefitVertices);
    * fillSingle:: // fillSecondaryVertex(in, out);
* datasets.json
    * LFV: preferred master version with "generatorWeight": 1.0

greyxray committed:
Fix compilation error

jd550179 committed:
automatic splitting caused missing files with n-x ids, should be added now

jd550179 committed:
now possible to have more then 10 resubmits

jd550179 committed:
first look if 0-x is in jobids if not do check the old way

hsert committed:
generatorWeight: the negative events on LO sample are taken into account after a recommendation from the generator group

hsert committed:
generatorWeight: a fix to the previous commit

hsert committed:
temporary changes to store 2016 training of tauID for 2017 samples

hsert committed:
a fix for the file creation using automatic splitting

hsert committed:
more fixes to automatic splitting fil list creation

greyxray committed:
dataset.json: Add missing 2017 saples

2017 rereco v1
relevant for MSSM
em, tt channel

jd550179 committed:
added btag discriminator at the right place

jd550179 committed:
added btag discriminator at the right place

hsert committed:
Merge remote-tracking branch 'origin/dictchanges_CMSSW94X' into dictchanges_CMSSW94X

hsert committed:
Merge branch 'master' into dictchanges_CMSSW94X

hsert committed:
Revert "temporary changes to store 2016 training of tauID for 2017 samples"

This reverts commit 5df9f4f1bdfdec871f975aa592b77d29e94c881a.

thomas-mueller reviewed (changes requested):
I will send my minor comments in mattermost. Here I just want to point out, that we need a check, that we can still use the existing 2016/2017 skims with the Artus/Kithiggs master (probably after a merge of the dictchanges branches there) or, if this does not work, a complete re-skim with the new dataformat in Kappa. If we are sure, that we can run Artus after the merge for ongoing analyses as before, we can go towards a merge of this PR.

hsert committed:
verbosity levels are changed to include higher levels as well

hsert committed:
checkout script for CMSSW94X-949 and for SL7 is updated

jd550179 committed:
Âadded lhe info for 2017 in the same way as 2016

jd550179 committed:
Âmet and electron smearing as done by kit

jd550179 committed:
added mcv2 april files

jd550179 committed:
added signal jhu, check cros-sec. and data 31March2018

jd550179 committed:
Âsingle top + tag to data: skim_data31March

jd550179 committed:
ewkz samples added

jd550179 committed:
EENoisePatch implementation

Vhajahmad committed:
added MuonEG 31march 2017 data

jd550179 committed:
mm jhu sample 2017

azotz committed:
updating run2 config to match electron id tag changes in 2017 config

jd550179 committed:
update recipe to the met corrections problems

jd550179 committed:
Merge remote-tracking branch 'origin/dictchanges_CMSSW94X' into dictchanges_CMSSW94X

jd550179 committed:
added the right globalTag for march31 data

jd550179 committed:
changed global tag to v15

jd550179 committed:
removed stuff which is already in CMSSW94x and larger

jd550179 committed:
also commented out in a very similar framework

jd550179 committed:
above 94 not 949

jd550179 committed:
changed tag to 15 and added sm and cp odd vbf jhu samples, maxmix not ready net

jd550179 committed:
added tagvalue to vbf jhu

jd550179 committed:
changed global tag from 15 to 16 and added madgraph H+2jet files

jd550179 committed:
saved npNLO, which is needed for stitching the madgraph h cp signal samples

jd550179 committed:
updated to latest in htt/sync from artur

jd550179 committed:
global tag 16 has jet problems according to IC, use 15

jd550179 committed:
changes as proposed on gitlab

jd550179 committed:
added backgrounds and signals and used latest xsec

jd550179 committed:
bugfix in kskimming + chechout cmssw9411_patch2

jd550179 committed:
removed deep tau ids

jd550179 committed:
remove deeptau data from cmsssw, 150 mb to large for crab

jd550179 committed:
additional files + dwinterb check to add RunIIFall17MiniAODv2_ to nickname

jd550179 committed:
added minor missing things in skim

jd550179 committed:
generatorweights

jd550179 committed:
generatorweights for all MC, gluglu sample had missing block, leading to missing number of events?

jd550179 committed:
fix in nicknaming of CP samples, generatorweights added for cp samples, rho corrected electrons effective area now takes the area and not the radius

azotz commented:
any updates on the status?

azotz committed:
Merge branch 'master' into dictchanges_CMSSW94X

azotz committed:
datasets.json: added file lists for embedding 2017

azotz committed:
kSkimming_2017_cfg.py: changed config for embedding 2017

azotz committed:
kSkimming_2017_cfg.py: added something for l1taus for embedding 2017

azotz committed:
Added Daniel's samples for GluGluHToMaxMixTauTau and
VBFHToPseudoscalerTauTau

azotz committed:
switched to official egamma repo

lucaswiens committed:
Fix a typo

lucaswiens committed:
Enable refitted Vertex for CMSSW version 9

lucaswiens committed:
Implement a workaround to make hashes compatible with Artus

The boost library of Kappa and Artus seems to be different, thus the hashes of
the individual leptons are saved and later combined in Artus.

azotz committed:
updated global tags for data, embedding and MC

data/embedding: globalTag: 94X_dataRun2_v10 --> 94X_dataRun2_v11

MC: globalTag: 94X_mc2017_realistic_v15 --> 94X_mc2017_realistic_v17

added a 'hack' to have private samples produced by 'adow' be treated
like official samples (much like the workaround for 'dwinterb's
samples)

lucaswiens committed:
Fix a typo

lucaswiens committed:
Implement a function to call the dbs by nickname

When submitting a task via crabConfig.py, it would try to generate the
config.Data.inputDataset using a function that did not provide the
correct inputDataset but just the nick (which is given anyway). This
fixes a bug where you could not submit a recovered task.

azotz committed:
added Tau 2017 Data sets

azotz committed:
updated tags for tau 2017 data

lucaswiens committed:
Delete an unused variable

The selection hash is now combined in artus, thus the
leptonSelectionHash variable is not needed anymore and the
leptonSelectionHash1 and leptonSelectionHash2 are used instead.

azotz committed:
DataFormats/interface/KTrack.h: added functions to get track parameters

azotz committed:
Merge branch 'dictchanges_CMSSW94X' of github.com:KappaAnalysis/Kappa into dictchanges_CMSSW94X

mbluj committed:
fix logic error

azotz committed:
Merge pull request #59 from mbluj/dictchanges_CMSSW102X

fix logic error

azotz committed:
added new sync VBF sync sample for 2017

and adjusted DatasetsManager to put "new_pmx" in the scenario if it is
part of the DBS name

Vhajahmad committed:
LFVZ4JetsToLLM50_madgraph-pythia8_13TeV  set added
